packages/hardhat-zksync-chai-matchers/src/constants.ts

```diff
@@ -0,0 +1 @@
+export const PLUGIN_NAME = '@matterlabs/hardhat-zksync-chai-matchers';
```

Comment:
Nitpick, but normally we add newlines at the end of the file. Please do so to be consistent with the company code style.

Reply:
@popzxc do we have lint config files that could (should) be used in all projects?

Reply:
> @popzxc do we have lint config files that could (should) be used in all projects?

Found it.